addons/mail/static/src/components/message_list/message_list.xml

```diff
@@ -6,6 +6,15 @@
             <t t-if="threadView">
                 <!-- No result messages -->
                 <t t-if="threadView.nonEmptyMessages.length === 0">
+                    <t t-if="threadView.thread.activities.length or threadView.threadViewer and threadView.threadViewer.chatter and threadView.threadViewer.chatter.isAttachmentBoxVisible">
```

Comment:
MessageList should not be aware of chatter concerns.

The best way to resolve this would be to add a new props in MessageList to control the state of the separator, props that will be false by default, but set true by Chatter (through ThreadView) when needed.

Note that probably the separator should actually not be inside the message list, but between MessageList and its sibling on its parent component ThreadView?
The reasoning for this that it makes no sense for a component itself to handle a separator between him and siblings, that should be the job of the parent.
However the counter argument would be if we considered this a header for the message list instead of a separator, in which case it would make sense to have it inside. For example for activity the "separator" is actually considered a "title" instead.

cc @alexkuhn any idea what would we want here?

Reply:
This separator should not be in `MessageList`. It should be in `ThreadView`, just before `<MessageList>` in the template.